src/graphics/vertex-format.js

```diff
@@ -186,37 +190,23 @@ class VertexFormat {
      */
     static _defaultInstancingFormat = null;
 
-    /**
-     * Initialize a default format use for instanced rendering.
-     *
-     * @param {GraphicsDevice} graphicsDevice - The graphics device used to create the format.
-     * @ignore
-     */
-    static init(graphicsDevice) {
-        const formatDesc = [
-            { semantic: SEMANTIC_ATTR12, components: 4, type: TYPE_FLOAT32 },
-            { semantic: SEMANTIC_ATTR13, components: 4, type: TYPE_FLOAT32 },
-            { semantic: SEMANTIC_ATTR14, components: 4, type: TYPE_FLOAT32 },
-            { semantic: SEMANTIC_ATTR15, components: 4, type: TYPE_FLOAT32 }
-        ];
-
-        VertexFormat._defaultInstancingFormat = new VertexFormat(graphicsDevice, formatDesc);
-    }
-
     /**
      * The {@link VertexFormat} used to store matrices of type {@link Mat4} for hardware instancing.
      *
      * @type {VertexFormat}
      */
     static get defaultInstancingFormat() {
```

Comment:
Why is the `get` needed here, BTW? I know it was like that before, but was curious.

Reply:
I'm not sure I understand what you're asking? The get is there to make it a property getter and not a function, so it can be called like this: `pc.VertexFormat.defaultInstancingFormat`

Reply:
Oh right, of course. Duh. 😆